src/init/application/templates/polymer-2.x/test/_element/_element_test.html

```diff
@@ -22,7 +22,7 @@
       suite('<%= elementName %>', function() {
         test('instantiating the element works', function() {
           var element = fixture('basic');
-          assert.equal(element.is, '<%= elementName %>');
+          assert.equal(element.constructor.is, '<%= elementName %>');
```

Comment:
I think there may be some issues with `constructor` when compiled to ES5?

How about something like:

```javascript
assert.true(!!customElements.get('<%= elementName %>'));
assert.true(element instanceof Polymer.Element);
```